pkgs/by-name/dh/dhcpig/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhcpig";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "kamorin";
+    repo = "DHCPig";
+    rev = version;
+    hash = "sha256-MquLChDuJe3DdkxxKV4W0o49IIt7Am+yuhdOqUqexS8=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+    scapy
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    scapy
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -R pig.py $out/bin/dhcpig
+  '';
+
+  meta = with lib; {
+    description = "DHCP exhaustion script written in python using scapy network library";
+    homepage = "https://github.com/kamorin/DHCPig";
+    license = licenses.gpl2;
```

`gpl2` is deprecated; Project's README says "GPL v2 or later" so I think this should be:
```suggestion
    license = licenses.gpl2Plus;
```